src/wasi/WASI.h

```diff
@@ -45,13 +45,21 @@ class WASI {
     F(random_get, I32I32_RI32)                     \
     F(fd_write, I32I32I32I32_RI32)                 \
     F(fd_read, I32I32I32I32_RI32)                  \
+    F(fd_readdir, I32I32I32I64I32_RI32)            \
     F(fd_close, I32_RI32)                          \
     F(fd_fdstat_get, I32I32_RI32)                  \
-    F(fd_prestat_dir_name, I32I32I32_RI32)         \
+    F(fd_fdstat_set_flags, I32I32_RI32)            \
     F(fd_prestat_get, I32I32_RI32)                 \
+    F(fd_prestat_dir_name, I32I32I32_RI32)         \
```

Comment:
I moved this definition so functions follow the order of the specifications so it's a bit easier to search.